### ISSUE TITLE
Syntax error in Parameter X-ES-REGISTRANT-TIPO-IDENTIFICACION

### ISSUE THREAD

**Issue by andp97-netsons** [hexonet/whmcs-ispapi-registrar#207]
**Describe the bug**
We see this error when try to register .es domain:
```
Domain Registration Failed - Domain ID: **** - Domain: *******.es - Error: Syntax error in Parameter X-ES-REGISTRANT-TIPO-IDENTIFICACION (Otra; For non-spanish owner)
```
Currently `.es` additonal domain fields:
```php
$additionaldomainfields[".es"][] = array(
                 "Name" => "Registrant Type",
                 "Type" => "dropdown",
         "Options" => ",Otra; For non-spanish owner,NIF/NIE; For Spanish Individual or Organization,Alien registration card",
                 "Required" => true,
                 "Ispapi-Name" => "X-ES-REGISTRANT-TIPO-IDENTIFICACION",
                 "Ispapi-Options" => ",0,1,3",
);
```

**EDIT:** Domain was manually registered on Hexonet portal with same data. 

**To Reproduce**
1. Register .es domain with Registrant Type: Otra

**Expected behavior**
Domain registration

**Screenshots**
<img width="600" alt="Schermata 2021-08-12 alle 10 13 01" src="https://user-images.githubusercontent.com/80452359/129162288-3831fe08-7200-423b-9ee6-49e265b27b85.png">

**Context (please complete the following information):**
 - WHMCS Version [8.0.5]
 - PHP Version [7.3]
 - ISPAPI Registrar Module Version [6.4.3]

**Comment by papakai** [comment]
Thanks for reporting @andp97-netsons. Would you mind forwarding the Module Log Entry for Command AddDomain? If you don't have the module log turned on, please forward the domain name to my hands by mail (kschwarz at hexonet dot net).

**Comment by papakai** [comment]
In addition: additionalfields.php is no longer required since v6 of our registrar module. Please cleanup all configuration entries in there related to hexonet/ispapi. (identify them by Ispapi-Name)

**Comment by andp97-netsons** [comment]
Email sent! 👍

**Comment by papakai** [comment]
Hi @andp97-netsons,

the root cause of this issue, is a mixup of the additionalfields.php configuration and how we do it in v6.
Please cleanup the additionalfields.php as mentioned before.

I've rolled out a fix for .ES that I noticed while testing. Find it released as [v6.6.0](https://github.com/hexonet/whmcs-ispapi-registrar/raw/master/whmcs-ispapi-registrar-latest.zip). This issue did not affect you in this case, but will do in future cases, so worth to upgrade.

You have probably to reorder that domain name then from scratch.
Sorry for the inconveniences caused.

Best
Kai

**Comment by papakai** [comment]
Just one thing to mention: if you are offering a Trustee/Local Presence Service (X-<TLD>-ACCEPT-TRUSTEE-TAG) like for .de, please keep such additional fields, but all others of that TLD, you can cleanup.
